conseil-lorre/src/test/scala/tech/cryptonomic/conseil/indexer/tezos/TezosDatabaseOperationsTestFixtures.scala

```diff
@@ -41,10 +41,10 @@ trait TezosDatabaseOperationsTestFixtures extends RandomGenerationKit {
 
   /* randomly generates a number of accounts with associated block data */
   def generateAccounts(
-      howMany: Int,
-      blockHash: BlockHash,
-      blockLevel: Int,
-      time: Instant = testReferenceTimestamp.toInstant
+                        howMany: Int,
+                        blockHash: TezosBlockHash,
+                        blockLevel: Int,
+                        time: Instant = testReferenceTimestamp.toInstant
```

Comment:
Why does formatting get nuts at times?

Reply:
That's good question, cause I did `sbt scalafmt` before asking for review. Let me check this out again.

Reply:
`sat scalafmt` did nothing with that. Only IntelliJ + Reformat Code shortcut did - weird.